### PR TITLE
WiP metadata endpoint rebased against develop

### PR DESCRIPTION
**Pull request by mcovarr**
Working to get something stable enough for Blue to build on in Sprint 12.

### CODE REVIEW

**Comment by geoffjentry** [comment]
Would it make sense for me to add that call start/end stuff in here?

**Comment by mcovarr** [comment]
Yes!  I thought that was already on develop.

**Comment by geoffjentry** [comment]
Yeah, that'd make sense wouldn't it?

File: src/main/scala/cromwell/engine/workflow/WorkflowManagerActor.scala [hunk 1]
```diff
@@ -184,6 +188,32 @@ class WorkflowManagerActor(dataAccess: DataAccess, backend: Backend) extends Act
     } yield callToLogsMap
   }
 
+  private def workflowMetadata(id: WorkflowId): Future[WorkflowMetadataResponse] = {
+    for {
+      callStandardStreamsMap <- workflowStdoutStderr(id)
```
Comment (geoffjentry) [inline src/main/scala/cromwell/engine/workflow/WorkflowManagerActor.scala]: Not a complaint but curious why this is called `Streams`. I thought it had to do with the data type but that doesn't seem to be the case

Comment (mcovarr) [inline src/main/scala/cromwell/engine/workflow/WorkflowManagerActor.scala]: https://en.wikipedia.org/wiki/Standard_streams

Comment (geoffjentry) [inline src/main/scala/cromwell/engine/workflow/WorkflowManagerActor.scala]: damn polymorphism

**Comment by scottfrazer** [comment]
Well :+1: for me

**Pull request merged**